Renaming (#3)
giantswarm/rbac-bootstrap-app: README.md

[![CircleCI](https://circleci.com/gh/giantswarm/rbac-bootstrap-app.svg?style=shield)](https://circleci.com/gh/giantswarm/rbac-bootstrap-app)

# RBAC bootstrap chart

> Giant Swarm offers a RBAC bootstrap App as a configuration package to bootstrap the initial role bindings.

Reviewer: ```suggestion
Giant Swarm offers the RBAC bootstrap app as a configuration package to bootstrap the initial role bindings for a workload cluster.
```

Here we define the rbac chart with its templates and default configuration.

**What is this app?**

- You can configure the list of users and groups that will have access to the cluster initially.

**Why did we add it?**

- Some customers want to define a set of bindings as a base to every cluster in order to enable their automation and admin to get quick access to the workload cluster API.

**Who can use it?**

> - This app run in all clusters and versions.

Reviewer: ```suggestion
- This app can be used in all clusters and release versions.
```

**This is an example of a values file you could upload using our web interface.**

> ```

Reviewer: ```suggestion
```yaml
```

# values.yaml
bindings:
  - role: edit
    users:
      - editor1@myco.com
      - editor2@myco.com
    groups:
      - devops
    namespaces:
      - ns1
      - ns2
  - role: admin
    users:
      - admin@myco.com
    groups:
      - adminteam
```

> This configuration create role bindings for `editor1@myco.com` and `editor1@myco.com` in the `ns1` and `ns2` namespaces. At the same time it creates a cluster role binding for `admin@myco.com` and `adminteam` to `admin` cluster role.

Reviewer: ```suggestion
This configuration creates role bindings for `editor1@myco.com` and `editor1@myco.com` in the `ns1` and `ns2` namespaces. At the same time it creates a cluster role binding for `admin@myco.com` and `adminteam` to the `admin` cluster role.
```
